libs/content-search-toolkit/src/queries/search.ts

```diff
@@ -33,21 +33,34 @@ export const searchQuery = (
   const mustNot: TagQuery[] = []
   let minimumShouldMatch = 1
 
-  should.push({
-    multi_match: {
-      fields: [
-        'title^6', // note boosting ..
-        'title.stemmed^2', // note boosting ..
-        'title.compound',
-        'content',
-        'content.stemmed',
-      ],
-      query: queryString,
-      fuzziness: 'AUTO',
-      operator: 'and',
-      type: 'best_fields',
-    },
-  })
+  const fieldsWeights = [
+    'title^6', // note boosting ..
+    'title.stemmed^2', // note boosting ..
+    'title.compound',
+    'content',
+    'content.stemmed',
+  ]
+  // * wildcard support for internal clients
+  if (queryString.trim() == '*') {
```

Comment:
=== is more consistent behaviour than ==

Reply:
on it